environment.yml

```diff
@@ -43,4 +43,4 @@ dependencies:
   - zlib
   - zstandard
   - conda-forge::eventio>=1.1.1
-  - ctapipe-extra>=0.3
+  - ctapipe-extra>=0.3.1
```

Comment:
You also need to change this in `setup.py` or we merge the download branch first

Reply:
done